pluto_build/01_dataloading_minor.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+source bin/config.sh
+
+# DROP all tables
+if [[ $1 == "drop" ]]; then
+    psql $BUILD_ENGINE -c "
+    DO \$\$ DECLARE
+        r RECORD;
+    BEGIN
+        FOR r IN (SELECT tablename FROM pg_tables WHERE schemaname = 'public' and tablename !='spatial_ref_sys') LOOP
+            EXECUTE 'DROP TABLE IF EXISTS ' || quote_ident(r.tablename) || ' CASCADE';
+        END LOOP;
+    END \$\$;
+    "
+fi
+
+# import_public PTS and CAMA from data library
+import_public pluto_pts 20230130 &  
+import_public pluto_input_cama_dof 20221130 &
+import_public pluto_input_numbldgs 20230130 &
+import_public pluto_input_geocodes 20230130 &
+
+# import_public spatial bounaries from data library
+import_public dcp_edesignation & #scheduled update?
+import_public dcp_cdboundaries_wi $VERSION_MAJOR_RELEASE &
```

Comment:
@mbh329 I'm curious about `VERSION_MAJOR_RELEASE`. since it isn't in `version.env`, datasets like this one pull the latest version. here's [a link to an example](https://github.com/NYCPlanning/db-pluto/actions/runs/4047194404/jobs/6960924898#step:6:60) in the CI data build I just started

there's also `MAJOR_RELEASE_VERSION=22v3` in `version.env` and it's unused. if it were used in places where `VERSION_MAJOR_RELEASE` is, it'd fail since many input datasets use versions like `22c`

is that on purpose?

Reply:
that was my oversight, thanks for catching